components/password_reset_form/password_reset_form.tsx

```diff
@@ -13,7 +13,7 @@ import LocalizedInput from 'components/localized_input/localized_input';
 import {t} from 'utils/i18n.jsx';
 
 interface Props {
-    location: {search: string };
+    location: {search: string;};
```

Comment:
yeah, you were right about the semicolon breaking the eslint ... it sometimes slips my mind that we do not add them on single line declarations. Sorry for the confusion!

```suggestion
    location: {search: string};
```